sales.js

```diff
@@ -28,4 +28,25 @@ for(var i = 0; i < allLocations.length; i++){
   var totalListItem = document.createElement('li');
   newList.appendChild(totalListItem);
   totalListItem.textContent = 'total';
-};
+//// tinkering with sum array function posted in slack. I think I want it to sum the varriable customerCount on line 20.
+    var testArray = customerCount;
```

Comment:
You don't need a variable called `testArray`; you can pass in whatever array you want to sum (e.g. `currentLocation.customersPerHour`).